SimCalorimetry/HcalTrigPrimProducers/python/hcaltpdigi_cff.py

```diff
@@ -34,8 +31,14 @@
 
 HcalTrigTowerGeometryESProducer = cms.ESProducer("HcalTrigTowerGeometryESProducer")
 
-run2_HCAL_2018.toModify(CaloTPGTranscoder, linearLUTs=cms.bool(True))
-run2_HCAL_2018.toModify(HcalTPGCoderULUT, linearLUTs=cms.bool(True))
-run3_common.toModify(HcalTPGCoderULUT, applyFixPCC=cms.bool(True))
-pp_on_AA.toModify(CaloTPGTranscoder, FG_HF_thresholds = cms.vuint32(15, 19))
-pp_on_AA.toModify(HcalTPGCoderULUT, FG_HF_thresholds = cms.vuint32(15, 19))
+from Configuration.Eras.Modifier_run2_HCAL_2018_cff import run2_HCAL_2018
+run2_HCAL_2018.toModify(HcalTPGCoderULUT, linearLUTs=True)
+
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(HcalTPGCoderULUT, applyFixPCC=True)
+
```

Comment:
```suggestion
```
By the way this is not needed, because the same modification is centralized in CalibCalorimetry.HcalPlugins.Hcal_PCCUpdate_cff, from where [L21](https://github.com/cms-sw/cmssw/blob/88ed27083405b16ee2be5509b57a291d1255f20b/SimCalorimetry/HcalTrigPrimProducers/python/hcaltpdigi_cff.py#L21) takes this parameter

Reply:
Removed as suggested